models/models.go

```diff
@@ -153,6 +153,16 @@ func getEngine() (*xorm.Engine, error) {
 		engine.Dialect().SetParams(map[string]string{"DEFAULT_VARCHAR": "nvarchar"})
 	}
 	engine.SetSchema(setting.Database.Schema)
+	if setting.Database.UsePostgreSQL && len(setting.Database.Schema) > 0 {
+		// Add the schema to the search path
+		if _, err := engine.Exec(`SELECT set_config(
+			'search_path',
+			? || ',' || current_setting('search_path'),
```

Comment:
Not familar with this syntax. Can you explain what this line does?

Reply:
`||` is (postgre?)sql for string concatenation

Reply:
SQL sure has weird operators. Maybe `CONCAT()` would be more clear?

Reply:
|| is standard ANSI SQL.

Reply:
Fine, whatever 😉